RecoTracker/LSTCore/interface/Module.h

```diff
@@ -12,147 +12,7 @@ namespace SDL {
-    objectRangesBuffer(unsigned int nMod, unsigned int nLowerMod, TDevAcc const& devAccIn, TQueue& queue)
-        : hitRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          hitRangesLower_buf(allocBufWrapper<int>(devAccIn, nMod, queue)),
-          hitRangesUpper_buf(allocBufWrapper<int>(devAccIn, nMod, queue)),
-          hitRangesnLower_buf(allocBufWrapper<int8_t>(devAccIn, nMod, queue)),
-          hitRangesnUpper_buf(allocBufWrapper<int8_t>(devAccIn, nMod, queue)),
-          mdRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          segmentRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          trackletRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          tripletRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          trackCandidateRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          quintupletRanges_buf(allocBufWrapper<int>(devAccIn, nMod * 2, queue)),
-          nEligibleT5Modules_buf(allocBufWrapper<uint16_t>(devAccIn, 1, queue)),
-          indicesOfEligibleT5Modules_buf(allocBufWrapper<uint16_t>(devAccIn, nLowerMod, queue)),
-          quintupletModuleIndices_buf(allocBufWrapper<int>(devAccIn, nLowerMod, queue)),
-          quintupletModuleOccupancy_buf(allocBufWrapper<int>(devAccIn, nLowerMod, queue)),
-          miniDoubletModuleIndices_buf(allocBufWrapper<int>(devAccIn, nLowerMod + 1, queue)),
-          miniDoubletModuleOccupancy_buf(allocBufWrapper<int>(devAccIn, nLowerMod + 1, queue)),
-          segmentModuleIndices_buf(allocBufWrapper<int>(devAccIn, nLowerMod + 1, queue)),
-          segmentModuleOccupancy_buf(allocBufWrapper<int>(devAccIn, nLowerMod + 1, queue)),
-          tripletModuleIndices_buf(allocBufWrapper<int>(devAccIn, nLowerMod, queue)),
-          tripletModuleOccupancy_buf(allocBufWrapper<int>(devAccIn, nLowerMod, queue)),
-          device_nTotalMDs_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)),
-          device_nTotalSegs_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)),
-          device_nTotalTrips_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)),
-          device_nTotalQuints_buf(allocBufWrapper<unsigned int>(devAccIn, 1, queue)) {
-      alpaka::memset(queue, hitRanges_buf, 0xff);
-      alpaka::memset(queue, hitRangesLower_buf, 0xff);
-      alpaka::memset(queue, hitRangesUpper_buf, 0xff);
-      alpaka::memset(queue, hitRangesnLower_buf, 0xff);
-      alpaka::memset(queue, hitRangesnUpper_buf, 0xff);
-      alpaka::memset(queue, mdRanges_buf, 0xff);
-      alpaka::memset(queue, segmentRanges_buf, 0xff);
-      alpaka::memset(queue, trackletRanges_buf, 0xff);
-      alpaka::memset(queue, tripletRanges_buf, 0xff);
-      alpaka::memset(queue, trackCandidateRanges_buf, 0xff);
-      alpaka::memset(queue, quintupletRanges_buf, 0xff);
-      alpaka::memset(queue, quintupletModuleIndices_buf, 0xff);
-      alpaka::wait(queue);
-    }
-  };
-
-  struct modules {
+  struct Modules {
```

Comment:
Do you think these non-templated structs (with associated functions) could end up causing symbol conflicts?

Reply:
renaming wouldn't affect that.
It should be possible to check with `nm` or `objdump`.
There are no out-of-line methods.

Reply:
yeah, I know it wouldn't affect it, but it could be changed now since those are already being modified. Let's just leave it for later.